monkey/tests/unit_tests/monkey_island/cc/repository/test_mongo_machine_repository.py

```diff
@@ -232,6 +232,25 @@ def test_get_machine_by_ip__retrieval_error(error_raising_machine_repository):
         error_raising_machine_repository.get_machines_by_ip("1.1.1.1")
 
 
+def test_get_machines(machine_repository):
+    retrieved_machines = machine_repository.get_machines()
+
```

Comment:
Also check that lengths are equal. As written, this won't test if `retrieved_machines` has extra machines in it.

Reply:
Can't we just assert the two lists?